Node ILP SDK demo

nkramer44 opened the pull request:
## High Level Overview of Change

Added a demo of new ILP functionality in Xpring-JS

### Context of Change

<!--
Please include the context of a change.
If a bug fix, when was the bug introduced? What was the behavior?
If a new feature, why was this architecture chosen? What were the alternatives?
If a refactor, how is this better than the previous implementation?

If there is a design document for this feature, please link it here.
-->

There weren't any demos for ILP in XpringSDK, so now we have one for node!

### Type of Change

<!--
Please check relevant options, delete irrelevant ones.
-->

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)

## Test Plan

Added a test run to circleCI for the new demo.

<!--
## Future Tasks
For future tasks related to PR.
-->

nkramer44 committed:
xpring-js ilp demo

nkramer44 committed:
add circle job

keefertaylor commented on node/src/index-ilp.js:
```diff
@@ -0,0 +1,32 @@
+const grpcUrl = 'hermes-grpc.ilpv4.dev'
+const demoUserId = "demo_user"
+const demoUserAuthToken = "2S1PZh3fEKnKg"
+async function main() {
```
nit: I'd add a blank line between your global vars and function definitions

keefertaylor reviewed (commented):
Could you:
- Update README.md to have another section / screenshot of this? 
- Rebase index -> index-xrp.js (and update CI / README.md)

keefertaylor commented:
It would also be great to update the root index.md in a sane way. Maybe just add a bullet at the bottom which enumerates the XRP functionality and says 'node only' in parenthesis? Open to whatever. I think our tech writers will take a look at this soon.

nkramer44 committed:
rebase index.js to index-xrp.js
rebase node-demo.png to node-xrp-demo.png
add ilp screenshot
update README.md at root and node/

Signed-off-by: nkramer44 <noah.ph.kramer@gmail.com>

nkramer44 committed:
update package-lock

nkramer44 commented:
@keefertaylor I updated the READMEs and renamed some files.  Also updated xpring-js to 1.5.1, just to keep it up to date with the latest version
